### PR TITLE
conduit-sprint gets meid as the input on query device

### PR DESCRIPTION
**Pull request by georgeismike**
conduit sprint expects MEID passed in for query device info, not device_serial_number.  Arguably this is a mistake in reactor. But for now I am going to have conduit-atom conform to conduit-sprint.

### CODE REVIEW

File: lib/conduit/atom/decorators/query_device_info_decorator.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 module Conduit::Atom::Decorators
-  class QueryDeviceInfoDecorator < Base
-    # Sends `device_serial_number` and `iccid` if present
-    def query_device_info_attributes
-      attributes = {}
-      attributes[:device_serial_number] = device_serial_number if device_serial_number
-      attributes[:iccid]                = iccid                if iccid
-      attributes
+    class QueryDeviceInfoDecorator < Base
```
Comment (mgates) [inline lib/conduit/atom/decorators/query_device_info_decorator.rb]: I think this in indented wrong.

File: lib/conduit/atom/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module Conduit
   module Atom
-    VERSION = '0.4.9'
+    VERSION = '0.4.10'
```
Comment (mgates) [inline lib/conduit/atom/version.rb]: I think this should be 0.5.0, it's a breaking change.

File: spec/conduit/atom/actions/query_device_info_spec.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@
   let(:query_device_attributes) do
     credentials.merge({
       carrier_id: 1,
-      device_serial_number: '111222333444555667',
+      meid: '111222333444555667',
```
Comment (mgates) [inline spec/conduit/atom/actions/query_device_info_spec.rb]: If you make this "123456789012345678" it's obvious at a glance how many digits it is.

Comment (georgeismike) [inline spec/conduit/atom/actions/query_device_info_spec.rb]: I like this idea.  I am going to leave it as is though. The length of this field has a couple different valid lengths.

**Comment by pedromartinez** [comment]
👍

**Pull request merged**